src/ResourceManager/Resources/Commands.Resources.Test/ScenarioTests/ActiveDirectoryTests.ps1

```diff
@@ -754,23 +804,64 @@ function Test-CreateDeleteSpPasswordCredentials
     Assert-AreEqual $cred2.Count 2
     $credCount = $cred2 | where {$_.KeyId -in $cred1.KeyId, $cred.KeyId}
     Assert-AreEqual $credCount.Count 2
+	$cred2 = $cred
+
+	# Add 1 key credential to the same app
+	$certCreated = 0
+	$cert = New-SelfSignedCertificate 
```

Comment:
Same comment as above - you can alternately provide a self-sgned cert as a test asset.

Reply:
How could I do that?

Reply:
You would provide a certificate file (.pef, .pfx) as part of the test project, use this in the test, and re-record, rather than requiring certificate creation in the test.  This will also require an exception in the CredScan suppressions file for the certificate (since it is only used for tests).